client/src/components/Onboarding/Onboarding.js

```diff
@@ -12,70 +17,71 @@ class Onboarding extends Component {
 			emails: []
```

Comment:
need joinToggle and createToggle in state? or am i missing something

Reply:
weirdly did not show up in GH but i see them in the code

Reply:
The tab was collapsed, just had to open it up